aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -69,12 +67,11 @@
 import software.amazon.awssdk.services.glue.model.TableInput;
 import software.amazon.awssdk.services.glue.model.UpdateDatabaseRequest;
 
-public class GlueCatalog extends BaseMetastoreCatalog implements Closeable, SupportsNamespaces, Configurable {
+public class GlueCatalog extends BaseMetastoreCatalog implements Closeable, SupportsNamespaces {
```

Comment:
Nit - Can you add a short doc comment that mentions the `HadoopConfigurableGlueCatalog`? E.g., I imagine that Spark users would want to possibly use the `HadoopConfigurableGlueCatalog` in many cases.